packages/picasso.js/src/web/text-manipulation/font-size-to-height.js

```diff
@@ -0,0 +1,25 @@
+const BASE = 24;
+const PAD = 4;
+const BUMP = 1e-12;
+const DEFAULT_HEIGHT = 16;
+const TEXT_REGEX = /^\s*([0-9]+|[0-9]+\.{1}[0-9]+)[px]{2}\s*$/i;
+
+function isValidFontSize(val) {
+  const type = typeof val;
```

Comment:
could write all of this as `return typeof value === 'string' ? TEST_REGEX.test(val) : false`